WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,13 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComURL
+{
+    NSRegularExpression *protocol = [NSRegularExpression regularExpressionWithPattern:@"wordpress\\.com/?$" options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
Updating, thanks!. NSURLComponents doesn't have any helper (*afaik*) that allows us to strip subdomains. 

Meaning that... `@"blog.wordpress.com"`'s *host* component will also carry the `blog` subdomain. I'll check, but we'll probably need the regex anyways =(